packages/commerce-sdk-react/src/hooks/useAccessToken.ts

```diff
@@ -27,6 +27,12 @@ interface AccessToken {
  */
 const useAccessToken = (): AccessToken => {
     const config = useConfig()
+
+    // TODO: Remove test logger before merging
+    if (config.logger) {
+        config.logger.info('config.logger.info test commerce-sdk-react message')
+    }
```

Comment:
NIT: In the future if we are going to be using the logger more, it might be nice to have a convenience hook to get the logger from the config. Food for thought, no action required now.

Reply:
I leave it for future consideration once we use the logger more.